README.md

```diff
@@ -45,10 +42,14 @@ It will print th cover page of the portfolio.
 
 In order to run this project you need: Any Browser e.g. Google Chrome
 
+### Live-Demo
+
+This is the live-demo of my Portfolio - https://tasnimchy96.github.io/portfolio.github.io/
```

Comment:
- [optional] great job. the live link works fine. you can shift it up under its main header. 
-  [optional] will advice you write the live demo this way 👇🏾 to make it more presentable
`[Live-Demo](https://tasnimchy96.github.io/portfolio.github.io/)`